docs/development/buildenvironments.rst

```diff
@@ -30,6 +30,8 @@ This would be a good place to start testing as the ``latest`` version could
 operate differently. See ``DOCKER_IMAGE`` below for setting this configuration
 option.
 
+.. note:: The size of the Docker images is around 5 to 9 GB.
```

Comment:
I think this is more useful before the `docker pull` command

Reply:
@stsewd Updated the PR. :+1: